examples/messaging/messages/createPush.cpp

```diff
@@ -0,0 +1,28 @@
+#include "Appwrite.hpp"
+#include "classes/Messaging.hpp"
+#include <iostream>
+
+int main() {
+    std::string projectId = "66fbb5a100070a3a1d19"; 
+    std::string apiKey = ""; 
+    Appwrite appwrite(projectId, apiKey); 
+  
+    std::string topicId = "67b3048600077f40b8a7"; 
+
+    Messaging messaging(projectId,apiKey);
```

Comment:
Initialisation of both Messaging and Appwrite is redundant. The messaging class inherits all properties of Appwrite parent. 

See how we do it other apis. 

Use `appwrite.getMesaaging().createPush()`